cms/src/components/content/ContentOverview.vue

```diff
@@ -131,6 +171,104 @@ router.currentRoute.value.meta.title = `${capitaliseFirstLetter(titleType)} over
             :buttonPermission="canCreateNew"
             data-test="no-content"
         /> -->
+        <div class="flex h-14 w-full gap-2 rounded-t-md bg-white p-2 shadow-lg">
+            <input
+                type="text"
+                class="h-full w-2/4 rounded-md border-[1px] border-zinc-200 p-4 text-sm focus:border-zinc-950 focus:bg-zinc-50 focus:outline-none focus:ring-0"
+                name="search"
+                placeholder="Search..."
+                data-test="search-input"
+                v-model="searchTerm"
+            />
+            <div>
+                <div class="relative flex h-full gap-1">
+                    <button
+                        class="flex h-full w-10 flex-row content-center justify-center rounded-md border-[1px] focus:border-black focus:outline-none"
+                        @click="() => (showSortOptions = true)"
+                        data-test="sort-toggle-btn"
+                    >
+                        <ArrowsUpDownIcon class="h-full w-4" />
+                    </button>
+                    <div
+                        ref="sortOptionsAsRef"
+                        class="absolute right-0 top-full mt-2 h-max w-40 rounded-lg border border-gray-300 bg-white p-2 shadow-lg"
+                        v-if="showSortOptions"
+                        data-test="sort-options-display"
+                    >
+                        <h4>Sort By:</h4>
+                        <div class="flex flex-col">
+                            <label>
+                                <input
+                                    class="ml-2 text-zinc-800 focus:border-0 focus:outline-none focus:ring-0"
+                                    type="radio"
+                                    name="sortOption"
+                                    value="title"
+                                    v-model="selectedSortOption"
+                                    data-test="sort-option-title"
+                                />
```

Comment:
I think an `LRadio` component would be nice so that we can easily reuse it in other places.

Reply:
As discussed, we could make this a follow-up ticket. Take the BCC component as an example: https://github.com/bcc-code/bcc-design/blob/main/design-library/src/components/BccRadio/BccRadio.vue